dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/ConnectionManager.java

```diff
@@ -1667,12 +1669,22 @@ public ResourceRefInfo getResourceRefInfo() {
      */
     final int registerXAResourceInfo(EmbeddableWebSphereTransactionManager tm,
                                      CommonXAResourceInfo xaResourceInfo, int commitPriority) {
+        return registerXAResourceInfo(tm, xaResourceInfo, commitPriority, null);
+    }
+
+    final int registerXAResourceInfo(EmbeddableWebSphereTransactionManager tm,
+                                     CommonXAResourceInfo xaResourceInfo, int commitPriority, String qmid) {
+        if (tc.isEntryEnabled())
+            Tr.entry(tc, "registerXAResourceInfo");
+        if (tc.isDebugEnabled()) {
+            Tr.debug(tc, "registerXAResourceInfo", tm, xaResourceInfo, commitPriority, qmid);
```

Comment:
Minor, but back-to-back trace statements are a waste.  Just put the debug args onto Tr.entry, and omit the Tr.debug entirely.